### PR TITLE
Development

### PR DESCRIPTION
**Pull request by skovbakke**
Added the option to configure DNS for the cluster.

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ A join token to use by other nodes joining the cluster. This is used to establis
 #### MASTER_CERTIFICATE_KEY
 A key used to encrypt the certificates.
```
Comment (jacobcrawford) [inline docs/config.md:36]: It is good practice to just have one optional variable in this case. 
If "CLUSTER_DNS" is set we obviously want to configure DNS, so there is no need for the "CONFIGURE_DNS" variable

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
Looks good, apart from a single change request.

File: docs/config.md (hunk 1 above)
Comment (skovbakke) [reply docs/config.md:36]: Should be fixed in the new commit

File: scripts/prepare_cluster.sh [hunk 2]
```diff
@@ -44,12 +51,15 @@ export NODE_NETWORK_INTERFACE=$NODE_NETWORK_INTERFACE
 export MASTER_TAINT=$MASTER_TAINT
 export NODE_GATEWAY_IP=$NODE_GATEWAY_IP
 export NODE_TYPE=master
+export CONFIGURE_DNS=$CONFIGURE_DNS
```
Comment (jacobcrawford) [inline scripts/prepare_cluster.sh:54]: This is no longer used 🗡️

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
kill it.

File: scripts/prepare_cluster.sh (hunk 2 above)
Comment (skovbakke) [reply scripts/prepare_cluster.sh:54]:
```suggestion
```
Yes!

**Review by jacobcrawford — APPROVED** [review]
congrats

**Pull request merged**